libraries/CMSIS-DSP.lib

```diff
@@ -1 +1 @@
-https://github.com/ARM-software/CMSIS-DSP/#3a04f817a4380c00ff51f9bcdc6c4e4bb7a18b80
\ No newline at end of file
+https://github.com/ARM-software/CMSIS-DSP/#72d6ccb7bf0ed5c7d3a0fb199d852d59ff31e85b
```

Comment:
Any reason why this is changed. The previous commit ID was correct. It should be:
https://github.com/ARM-software/CMSIS-DSP/#3a04f817a4380c00ff51f9bcdc6c4e4bb7a18b80

Reply:
Ok. i was getting compiler errors if i updated only no-os. i had to update the other projects too, i thought we should always be on the latest of eatch.   After i updated the sub modules to what is pointed by the current sha, there are no compiler errors, so i i can leave it as ti was.